packages/design-system/src/components/contextMenu/stories/index.js

```diff
@@ -389,6 +391,60 @@ export const QuickActionMenu = (args) => {
   );
 };
 
+export const HorizontalMenu = (args) => {
+  return (
+    <Grid>
+      <Container>
+        <Text>{'Dummy horizontal menu'}</Text>
+        <ContextMenu isHorizontal isInline isSecondary {...args}>
+          <MenuItems.MenuButton onClick={action('Clicked on bucket')}>
+            <MenuItems.MenuIcon
+              title="Bucket"
+              placement={TOOLTIP_PLACEMENT.BOTTOM}
+            >
+              <Bucket />
+            </MenuItems.MenuIcon>
+          </MenuItems.MenuButton>
+          <MenuItems.MenuButton onClick={action('Clicked on link')}>
```

Comment:
Hi! Just wanted to give you a heads up that `actions` are getting updated with the new Storybook controls. We now can define them within `argTypes` along with `args` at the top of the Storybook component. Everything has been upgraded so feel free to use the [new pattern](https://github.com/GoogleForCreators/web-stories-wp/pull/10274). The old `actions` have already been removed from the `Dashboard`, `WP-Dashboard` and `WP-Story-Editor` packages. The [Design-System](https://github.com/GoogleForCreators/web-stories-wp/pull/10520) and [Story-Editor](https://github.com/GoogleForCreators/web-stories-wp/pull/10499) packages will likely have the "old" `actions` removed later today or tomorrow.